Documentation of ifNotFinite function (#88)
BayoNet/ClickHouse: docs/en/query_language/functions/other_functions.md

Accepts Float32 and Float64 and returns UInt8 equal to 1 if the argument is infinite, otherwise 0. Note that 0 is returned for a NaN.

## ifNotFinite {#ifnotfinite}

> Accepts Float32 and Float64. The function has the [ternary operator](https://clickhouse.tech/docs/en/query_language/functions/conditional_functions/#ternary-operator) structure: `ifNotFinite(x, y) = isFinite(x) ? x : y`. It returns `x` if `x` if finite, and `y` if `x` is not finite. 

Reviewer: The function does something between input and output, this is the point of this description. For parameters and returned values you have two sections below.

What the function does? That is a question.

The function checks whether `x` is a finite value. That is the answer.

**Syntax** 

```
ifNotFinite(x,y)
```
**Parameters** 

> - `x` — Float32 or Float64

Reviewer: `x`  — Value to be checked for infinity. Type: [Float*](link to the floating point data type description).

> - `y` — Float32 or Float64

Reviewer: `y` — Fallback value. Type: [Float*](link to the floating point data type description).

**Returned value**

> - returns `x` if `x` if finite, and `y` if `x` is not finite

Reviewer: ```suggestion
- `x` if `x` if finite.
- `y` if `x` is not finite.
```

**Example**

Query:

```

> SELECT ifNotFinite(sqrt(-1), -42)

Reviewer: sqrt(-1) is not Infimum, it is NaN. 

This example `select 1/0 as infimum, ifNotFinite(infimum,42)` is more correct.

Accepts Float32 and Float64 and returns UInt8 equal to 1 if the argument is infinite, otherwise 0. Note that 0 is returned for a NaN.

## ifNotFinite {#ifnotfinite}

Checks whether `x` is a finite value.

**Syntax** 

```
ifNotFinite(x,y)
```
**Parameters** 

- `x` — Value to be checked for infinity. Type: [Float*](../../data_types/float.md).
- `y` — Fallback value. Type: [Float*](../../data_types/float.md).

**Returned value**

- `x` if `x` is finite.
- `y` if `x` is not finite.

**Example**

Query:

```
SELECT 1/0 as infimum, ifNotFinite(infimum,42)
```

Result:

```
┌─infimum─┬─ifNotFinite(divide(1, 0), 42)─┐
│     inf │                            42 │
└─────────┴───────────────────────────────┘
```

> Similar result can be made by [ternary operator](conditional_functions.md#ternary-operator) `isFinite(x) ? x : y`.

Reviewer: ```suggestion
You can get the similar result by [ternary operator](conditional_functions.md#ternary-operator) `isFinite(x) ? x : y`.
```
